konnectd/ui/build/static/js/identifier-app.cde47c8f.chunk.js

```diff
@@ -0,0 +1,2 @@
on(){return l})),o.d(n,"g",(function(){return u})),o.d(n,"h",(function(){return d})),o.d(n,"a",(function(){return k})),o.d(n,"i",(function(){return g})),o.d(n,"d",(function(){return p})),o.d(n,"l",(function(){return m}));var t="RECEIVE_ERROR",a="RESET_HELLO",s="RECEIVE_HELLO",r="RECEIVE_VALIDATE_LOGON",c="REQUEST_LOGON",i="RECEIVE_LOGON",l="UPDATE_INPUT",u="REQUEST_CONSENT_ALLOW",d="REQUEST_CONSENT_CANCEL",k="RECEIVE_CONSENT",g="REQUEST_LOGOFF",p="RECEIVE_LOGOFF",m="SERVICE_WORKER_NEW_CONTENT"},402:function(e,n,o){"use strict";o.r(n);var t=o(186),a=o.n(t),s=o(188),r=o(7),c=o.n(r),i=o(21),l=o.n(i),u=o(193),d=o.n(u),k=o(28),g=o(378),p=o(344),m=o(342),b=o(377),h=o(343),f=o(113),y=o(201),v=(o(202),o(34)),w=o(128),B=o.n(w).a.parse(document.location.search),E=B.flow||"";delete B.flow;var N={hello:null,error:null,flow:E,query:B,updateAvailable:!1,pathPrefix:function(){var e=document.getElementById("root"),n=e?e.getAttribute("data-path-prefix"):null;return n&&"__PATH_PREFIX__"!==n||(n="/signin/v1"),n}()};var S=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:N,n=arguments.length>1?arguments[1]:void 0;switch(n.type){case v.b:return Object.assign({},e,{error:n.error});case v.k:return Object.assign({},e,{hello:null});case v.c:return Object.assign({},e,{hello:{state:n.state,username:n.username,displayName:n.displayName,details:n.hello}});case v.l:return Object.assign({},e,{updateAvailable:!0});default:return e}},O=o(61);var H=function(){var e=arguments.length>0&&void 0!==arguments[0]?arguments[0]:{loading:"",username:"",password:"",errors:{}},n=arguments.length>1?arguments[1]:void 0;switch(n.type){case v.f:return Object.assign({},e,{errors:n.errors,loading:""});case v.g:case v.h:case v.j:return Object.assign({},e,{loading:n.type,errors:{}});case v.a:case v.e:return n.success?e:Object.assign({},e,{errors:n.errors?n.errors:{},loading:""});case v.d:return Object.assign({},e,{username:"",password:""});case v.m:return delete e.errors[n.name],Object.assign({},e,Object(O.a)({},n.name,n.value));default:return e}},D=Object(f.c)({common:S,login:H}),j=[y.a];var R=window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__||f.d,T=Object(f.e)(D,R(f.a.apply(void 0,j)));var K=function(e,n){return n.reduce((function(n,o){return n[o]=e[o],n}),{})}({de:o(205),"en-GB":{},"en-US":{},fr:o(206),hi:o(207),is:o(208),nb:o(209),nl:o(210),"pt-PT":o(211),ru:o(212)},["de","en-GB","en-US","fr","hi","is","nb","nl","pt-PT","ru"]),x=function(){var e=Object(s.a)(a.a.mark((function e(n){return a.a.wrap((function(e){for(;;)switch(e.prev=e.next){case 0:console.info("locale",n);case 1:case"end":return e.stop()}}),e)})));return function(n){return e.apply(this,arguments)}}(),A=d()({loader:function(){return Promise.all([o.e(9),o.e(5)]).then(o.bind(null,406))},loading:b.a,timeout:2e4});l.a.render(c.a.createElement(k.a,{store:T},c.a.createElement(g.a,{theme:p.a},c.a.createElement(m.b,{onLocaleChanged:x,messages:K},c.a.createElement(A,null)))),document.getElementById("root")),Object(h.a)()}}]);
```

Comment:
Codacy found an issue: [The numeric literal '2e4' will have at different value at runtime.](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6337760)